README.md

```diff
@@ -13,7 +13,7 @@
 
 ## Live Demo
 You can see my page live here: 
-[Check it out!](https://gabrielyea.github.io/module1-capstone/)
```

Comment:
- [ ] **OPTIONAL**
For a more good-looking README, it'd be great if you attach a screenshot of the app. 😄

Reply:
Fixed also!

Reply:
nicely done 👍